i_scene_cp77_gltf/main/bartmoss_functions.py

```diff
@@ -23,38 +35,39 @@ def rotate_quat_180(self,context):
         return{'FINISHED'}
     
     
+# deselects other objects and fully selects an object in both the viewport and the outliner
 def select_object(obj):
     for o in bpy.context.selected_objects:
         o.select_set(False)
     obj.select_set(True)
     bpy.context.view_layer.objects.active = obj
     
     
+## returns the volume of a given mesh by applying a rigid body with a material density of 1 and then returning the calculated mass
```

Comment:
should this delete the rigid body when its done the calc?

Reply:
Yeah, good thought.